intro/content/intro.tex

```diff
@@ -1,33 +1,133 @@
 \section{Einführung}
 
-\begin{frame}{Ziele}
+
+\begin{frame}{Motivation}
   \begin{center}
-    \textcolor{vertexDarkRed}{\Huge Auf das Praktikum vorbereiten} \\[3ex]
-    \Large Daten auswerten \hspace{2em} Plotten \hspace{2em} Fehlerrechnung
+    \textcolor{vertexDarkRed}{\Huge Auf das Praktikum vorbereiten} \\[\baselineskip]
+    \Large Daten:\hspace{2em} Abspeichern \hspace{2em} Auswerten \hspace{2em} Visualisieren \\ [\baselineskip]
+    Unsicherheiten bestimmen \hspace{2em} Protokoll verfassen\\
```

Comment:
I would change the second line to:
Zusammen arbeiten \hspace{2em} Protokoll verfassen
For me, uncertainties are part of the Auswertungs-part.

Reply:
Just a slight nuance, but "Zusammenarbeiten" would be better, as it implies working together on a project (e.g. to finish it together), whereas "zusammen arbeiten" only implies working together. The latter does not necessarily mean working on the same project.